src/main/steps/index.ts

```diff
@@ -143,7 +102,8 @@ const getStepsWithContent = (sequence: Step[], subDir = ''): StepWithContent[] =
   if (sequence?.length) {
     for (const step of sequence) {
       const { url } = parseUrl(step.url);
-      const stepDir = `${dir}${url.startsWith(subDir) ? url : `${subDir}${url}`}`;
+      const subdirurl = url.startsWith(subDir) ? url : `${subDir}${url}`;
```

Comment:
Why do we need subdir? what is the rationale behind it?

Reply:
sonar lint is working fine on that.